graftm/__init__.py

```diff
@@ -1,8 +1,4 @@
-""" graftm/__init__.py """
-
-# __all__ = []
-
-#from .graftm import *
+""" enrichm/__init__.py """
```

Comment:
not sure this is right

Reply:
not sure how this slipped through. Thanks